squid-blockexplorer/src/blocks/utils.ts

```diff
@@ -98,22 +98,18 @@ const MAX_PIECES_IN_SECTOR = 1000n;
 
 /**
  * Calculates the number of sectors pledged by a solution range, based on monorepo calculation
- * in function solution_range_to_sectors: https://github.com/subspace/subspace/blob/main/crates/subspace-runtime/src/lib.rs#L240-252
+ * in function solution_range_to_sectors: https://github.com/subspace/subspace/blob/main/crates/subspace-runtime/src/lib.rs#L227-L236
  * @param {bigint} solutionRange - range of the solution
  * @return {bigint} - sectors pledged number
  */
 export function solutionRangeToSectors(solutionRange: bigint): bigint {
-  const SIZE_OF_SOLUTION_RANGE = 8n;
-  const SCALAR_FULL_BYTES = 32n;
   const MAX_U64 = 2n ** 64n - 1n;
   const SLOT_PROBABILITY = [1n, 6n];
   const RECORD_NUM_CHUNKS = 32768n;
   const RECORD_NUM_S_BUCKETS = 65536n;
 
-  const auditChunksPerChunk = SCALAR_FULL_BYTES / SIZE_OF_SOLUTION_RANGE;
-  const sectors = MAX_U64
+const sectors = MAX_U64
```

Comment:
Looks a bit ugly with indentation removed:
```suggestion
  const sectors = MAX_U64
```